SYCL/Basic/cgeist/stream-triad.cpp

```diff
@@ -0,0 +1,47 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -Xcgeist -gen-all-sycl-funcs %s -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// REQUIRES: linux
+// UNSUPPORTED: hip || cuda
+
+#include <sycl/sycl.hpp>
+using namespace sycl;
+static constexpr unsigned N = 16;
+static constexpr int scalar = 8;
+
+template <typename T>
+void host_stream_triad(std::array<T, N> &A, std::array<T, N> &B, std::array<T, N> &C) {
```

Comment:
[nit]: The name of this function is strange because the `parallel_for` is actually meant to run on an accelerator device. 
How about just `stream_triad`?